test/hotspot/jtreg/containers/docker/TestPids.java

```diff
@@ -93,6 +93,17 @@ private static void checkResult(List<String> lines, String lineMarker, String ex
                 String[] parts = line.split(":");
                 System.out.println("DEBUG: line = " + line);
                 System.out.println("DEBUG: parts.length = " + parts.length);
+                if (expectedValue.equals("no_value_expected")) {
+                    Asserts.assertEquals(parts.length, 2);
+                    String ivalue = parts[1].replaceAll("\\s","");
+                    System.out.println("Found " + lineMarker + " with value: " + ivalue);
+                    try {
+                        int ai = Integer.parseInt(ivalue);
```

Comment:
Could you move the debug print line to after line 101, please. It could say:

`System.out.println("Found " + lineMarker + " with value: " + ai + ". PASS.");`

Reply:
the debug println has been moved down.